packages/terra-form-select/package.json

```diff
@@ -43,7 +43,7 @@
     "compile:clean": "rm -rf lib",
     "compile:build": "babel src --out-dir lib --copy-files",
     "lint": "npm run lint:js && npm run lint:scss",
-    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:js": "eslint --fix --ext .js,.jsx . --ignore-path ../../.eslintignore",
```

Comment:
nice - should we set the `fix` flag across all components then?

Reply:
oops. I mean to remove that.